pkg/controller/controller_broker_test.go

```diff
@@ -319,18 +319,21 @@ func TestReconcileClusterServiceBrokerRemovedClusterServiceClass(t *testing.T) {
 }
 
 // TestReconcileClusterServiceBrokerRemovedAndRestoredClusterServiceClass
-// validates where Service Catalog has a class that is marked as
-// RemovedFromBrokerCatalog but then the ServiceBroker adds the class back into
-// its getCatalog response.  This should result in the class's status being
-// updated resetting the RemovedFromBrokerCatalog to false.
+// validates where Service Catalog has a class and plan that is marked as
+// RemovedFromBrokerCatalog but then the ServiceBroker adds the class and plan
+// back into its getCatalog response.  This should result in the class's status
+// and plan's status being updated resetting the RemovedFromBrokerCatalog to
+// false.
 func TestReconcileClusterServiceBrokerRemovedAndRestoredClusterServiceClass(t *testing.T) {
```

Comment:
I think that this ought to have a separate test so that we don't conflate tests for two different scenarios into a single pass/fail.